packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -354,6 +355,12 @@ export interface IGCRuntimeOptions {
 	 */
 	sessionExpiryTimeoutMs?: number;
 
+	/**
+	 * Delay between when Tombstone should run and when the object should be deleted.
+	 * If not present, a default (non-zero) valua will be used.
+	 */
+	tombstoneSweepDelayMs?: number;
```

Comment:
Should we just call it `sweepDelayMs`? Since tombstone is an integral part of the GC stages, its natural that sweep will follow tombstone.

Reply:
I started with it like that, but felt like it could be misconstrued as the full sweep timeout (relative to unreferenced time).  It's supposed to be read as the "tombstone-sweep delay", i.e. the delay between tombstone and sweep.  Open to other suggestions of course.

Reply:
With `tombstoneSweepDelayMs`, my brain associates this with tombstone feature not sweep.